docs/api-reference.md

```diff
@@ -33,7 +33,7 @@ _Appears in:_
 
 
 
-KongClusterPlugin is the Schema for the  API.
+KongClusterPlugin is the Schema for the  API. The only differences between KongPlugin and KongClusterPlugin are that KongClusterPlugin is a Kubernetes cluster-level resource instead of a namespaced resource, and can be applied as a global plugin using `global` label.
```

Comment:
Can we add some comments to this file stating that it's generated by - insert here Makefile target name - and should not be modified by hand? This would ideally get into the doc sources at docs.konghq.com as well so that no one gets surprised because their changes got overwritten.

Reply:
Good point, ffe6603.